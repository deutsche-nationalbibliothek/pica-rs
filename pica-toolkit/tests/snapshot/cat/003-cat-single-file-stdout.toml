bin.name = "pica"
args = "cat math.dat.gz"
status = "success"
stderr = ""

