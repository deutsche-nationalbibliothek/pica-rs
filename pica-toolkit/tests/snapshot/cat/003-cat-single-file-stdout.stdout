001A 01250:01-07-88001B 09999:06-09-19t17:04:50.000001D 09999:06-09-19001U 0utf8001X 00002@ 0Tsz003@ 0040379442003U ahttp://d-nb.info/gnd/4037944-9004B asaz007K agnd04037944-9007N aswd04037944-9vzg008A asag008B awazao037G cT1--0151d2t2007-01-01037G c510d4t2007-01-01041@ aReine Mathematik041A aMathematik041P aMathematicsuhttp://lccn.loc.gov/sh85082139SDLC0sh 850821392lcshvMACS-Mapping. Bitte keine Änderungen vornehmen.041P aMathématiquesuhttp://data.bnf.fr/11932434SFrPBN0FRBNF1193243482ramvMACS-Mapping. Bitte keine Änderungen vornehmen.041R 90401594347Ts1VsazAgnd04015943-7aExakte Wissenschaften4obal042A a28047A/03 eDE-101047A/03 rDE-101047C SswdisaMathematik04037944-9050C aBearbeitungssperre (Top500) - Änderungswünsche an DNB050E aM070A/02 SMACS00000750
